src/compile/mark/mark.ts

```diff
@@ -60,12 +60,12 @@ function parsePathMark(model: UnitModel) {
   const details = detailFields(model);
 
   const role = model.markDef.role || markCompiler[mark].defaultRole;
-
+  const clip = model.markDef.clip != null ? model.markDef.clip : scaleClip(model);
```

Comment:
Since `model.markDef.clip` should be boolean, `null` is not technically a valid value. If the user specifies `null`, should we interpret that to be `false`?